infrastructure/tooling/src/generate/generators/docker-compose.js

```diff
@@ -445,11 +453,16 @@ exports.tasks.push({
         }
 
         dockerCompose.services[`${name}-${proc}`] = {
-          volumes: devVolumes,
+          volumes: [
+            ...devVolumes,
+            `./services/${name}:/app/services/${name}`, // service should only care about own code
+          ],
           environment: {
             LEVEL: 'info',
             DEBUG: 'taskcluster:*',
           },
+          command: '',
+          entrypoint: `/bin/sh -c "${procs[proc].command.replace(/^node /, `npx nodemon ${nodemonFlags(name)} `)}"`,
```

Comment:
This is arguable.. We can also change the base image and always install `nodemon` with dependencies, but it will increase slightly image and will add +1 dependency to the tree.
Otoh `npx nodemon` works perfectly by fetching, caching and running it, only adding small delay at first start.

Instead of `/entrypoint.sh` we use command directly and just replace `node` with `nodemon`